Cargo.toml

```diff
@@ -17,7 +17,7 @@ features = ["stm32f103", "rt"]
 cortex-m = "0.5.8"
 nb = "0.1.1"
 cortex-m-rt = "0.6.7"
-stm32f1 = "0.6.0"
+stm32f1 = "0.7.0"
```

Comment:
Uhm, can I borrow your crystal ball for a sec? 😅

Reply:
Of course not. You can break it. The art of seeing the future is passed down from generation to generation.

PS. See https://github.com/stm32-rs/stm32-rs/pull/116

Reply:
For testing you can use https://github.com/burrbull/stm32f1 until 0.7.0 be published.